wiki/Help_Centre/Website/id.md

```diff
@@ -0,0 +1,118 @@
+
+-->
+
+#### Memblokir pengguna-pengguna yang tidak ditambahkan sebagai teman untuk dapat mengirimkan pesan pribadi (*private message*) melalui forum
+
+Tuju laman [pengaturan akun](https://osu.ppy.sh/home/account/edit) dan aktifkan opsi `Blokir pesan pribadi dari orang yang tidak ada dalam daftar teman Anda` yang ada pada kolom Kebijakan Privasi. Ke depannya, kamu tidak akan lagi mendapatkan pesan forum yang berasal dari pengguna-pengguna yang tidak kamu tambahkan sebagai teman.
+
+### Apa yang harus saya lakukan apabila saya masih merasa tersinggung atau terlecehkan oleh pengguna-pengguna tertentu walaupun saya sudah memblokir mereka?
+
+**Jangan khawatir. Kami siap untuk membantumu.**
+
+Apabila kamu masih mendapatkan perlakuan yang kurang menyenangkan dari mereka terlepas dari segala upaya yang telah kamu lakukan sebelumnya, harap hubungi kami melalui email pada alamat [support@ppy.sh](mailto:support@ppy.sh).
+
+Pastikan kamu melampirkan *chat log* atau bukti-bukti lain yang mendukung kasusmu agar kami dapat menindaklanjuti para pelanggar yang bersangkutan sebagaimana mestinya.
+
+### Apa itu kolom “saya!” yang sering saya lihat pada laman profil milik para pengguna lain?
+
+**Kolom “saya!”, atau yang sering disebut juga dengan istilah *userpage*, merupakan kolom pada laman profil yang secara khusus tersedia bagi para pemilik [supporter tag](https://osu.ppy.sh/home/support). Kolom ini dapat kamu isi dengan hal apapun yang kamu inginkan - baik itu skor-skor yang pernah kamu torehkan, beatmap-beatmap milikmu, atau apapun itu!**
+
+Para pemilik [supporter tag](https://osu.ppy.sh/home/support) dapat membuat kolom “saya!” pada laman profil milik mereka kapanpun yang mereka mau. Kolom ini bebas untuk diisi dengan hal apapun - selama tidak melanggar [peraturan komunitas](/wiki/Rules) yang berlaku, tentunya!
+
+Apabila kamu sebelumnya pernah membuat kolom “saya!” pada laman profil osu! milikmu, kolom tersebut **tidak akan pernah hilang dan akan tetap dapat kamu sunting secara bebas ke depannya** bahkan ketika kamu sudah tidak lagi memiliki [supporter tag](https://osu.ppy.sh/home/support) yang berlaku.
+
+### Konten-konten yang ada pada kolom “saya!” milik saya tiba-tiba menghilang!
+
+**Pada umumnya, hal ini disebabkan karena tim kami menemukan konten-konten pada kolom “saya!” milikmu yang melanggar [peraturan komunitas](/wiki/Rules) yang berlaku.**
+
+Apabila kamu berkeberatan dengan hal ini, kamu dapat membuka tiket layanan dukungan (*support ticket*) baru dengan mengirimkan email pada alamat [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+Apabila kamu tidak yakin apakah konten yang kamu miliki (atau yang hendak kamu letakkan) pada kolom “saya!” milikmu melanggar peraturan atau tidak, harap hubungi anggota-anggota [Global Moderation Team](/wiki/People/Global_Moderation_Team) untuk mendapatkan arahan dan konfirmasi lebih lanjut.
```

Comment:
```suggestion
Apabila kamu tidak yakin apakah konten yang kamu miliki (atau yang hendak kamu letakkan) pada kolom “saya!” milikmu melanggar peraturan atau tidak, harap hubungi anggota-anggota [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) untuk mendapatkan arahan dan konfirmasi lebih lanjut.
```

Reply:
o this error appears to also be persisting in the english version of the article fwiw, might worth fixing this later on a different pr ig

...but yeah for now i got this fixed in the localized article 👍

Reply:
I already did it on #5864